autosklearn/ensembles/abstract_ensemble.py

```diff
@@ -26,6 +26,18 @@ def fit(self, base_models_predictions, true_targets, model_identifiers):
         """
         pass
 
+    @abstractmethod
+    def get_train_score(self):
+        """ Reports the methods score during training.
+
+        Parameters
```

Comment:
if there are no parameters nor return values, the headlines are not needed, right?